### ISSUE TITLE
Add better descriptions for commands

### ISSUE THREAD

**Issue by tohanss** [repobee/repobee-sanitizer#125]
sort of connected to #120 but I think it should be separate as this is in the code and not the README 

Specifically we should update:
`--no-commit` 
`--target-branch`

but should look to make all the others look as good as possible

**Comment by tohanss** [comment]
@slarse I seriously don't remember exactly what `--no-commit` does

**Comment by slarse** [comment]
Instead of creating a commit. It doesn't. Just modifies the working tree.

**Comment by slarse** [comment]
I'd suggest that you update all of the command descriptions as well. At least `sanitize repo` is wonderfully uninformative:

```
repobee -p sanitizer sanitize repo -h
usage: repobee sanitize repo [-h] [--tb] [-q] [-r REPO_ROOT] [--force] (--no-commit | --target-branch TARGET_BRANCH)

Automatically identifies and sanitizes files in a git repository
```

**Comment by slarse** [comment]
The category description is even better:

```
epobee -p sanitizer sanitize -h
usage: repobee sanitize [-h] {repo,file} ...

Sanitize stuf more elaborately.
```

**Comment by tohanss** [comment]
Yep, thats what this issue is for!

**Comment by slarse** [comment]
@tohanss Don't forget this one as well :)

**Comment by tohanss** [comment]
Fixed as of #127
